public/js/discover/searchList.js

```diff
@@ -83,14 +83,17 @@ export function selectCard(id = null, scroll = false) {
  * @return {string} The formatted phone number.
  */
 function formatPhoneNumber(number) {
-  if (number.length == 9) {
+  if (number.length == 10) {
     const areaCode = number.substring(0, 3);
     const exchangeCode = number.substring(3, 6);
-    const lineNumber = number.substring(6, 9);
-    return `(${areaCode})-${exchangeCode}-${lineNumber}`;
-  } else if (number.length == 10) {
-    const countryCode = number.substring(0, 1);
-    return `+${countryCode} ${formatPhoneNumber(number.substring(1, 10))}`;
+    const lineNumber = number.substring(6, 10);
+    return `(${areaCode}) ${exchangeCode}-${lineNumber}`;
+  } else if (number.length > 10) {
+    const countryCode = number.substring(0, number.length - 10);
```

Comment:
I think the `countryCode` formatting logic is not ideal in this case, since if we want to support non-US country code, we should also format the rest of the number against that non-US country code.

Since currently we are only supporting services in US, could we try using regex to match and format the number? Example: https://stackoverflow.com/a/8358141